Description: Use a monospace font for the legend so the column alignment works
 after upgrading to RRDTool 1.3
Author: Matthias Schmitz <matthias@sigxcpu.org>
Bug-Debian: http://bugs.debian.org/499033
Forwarded: no
Index: munin-1.2.6/server/munin-graph.in
===================================================================
--- munin-1.2.6.orig/server/munin-graph.in	2008-10-15 21:57:26.000000000 +0200
+++ munin-1.2.6/server/munin-graph.in	2008-10-15 22:10:31.000000000 +0200
@@ -832,10 +832,17 @@
 	    next unless ($draw{$time});
 	    my @complete = ();
 	    if ($RRDkludge) {
-		push (@complete,
- 		      '--font' ,'LEGEND:7:@@LIBDIR@@/VeraMono.ttf',
+		# since rrdtool 1.3 with libpango the LEGEND column alignment 
+		# only works with monospace fonts
+	  	if ( $RRDs::VERSION >= 1.3 ) { 
+		    push (@complete,
+ 		      '--font' ,'LEGEND:7:monospace');
+		} else {
+		    push (@complete,
+ 		      '--font' ,'LEGEND:7:@@LIBDIR@@/VeraMono.ttf');
+		}
+		    push (@complete,
  		      '--font' ,'UNIT:7:@@LIBDIR@@/VeraMono.ttf',
-
  		      '--font' ,'AXIS:7:@@LIBDIR@@/VeraMono.ttf');
 	    }
 
